main.py

```diff
@@ -1,7 +1,47 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
+
+
+def get_firstname_lastname_from_full_name(fullname: str) -> dict:
+    data = fullname.split()
+    return {"first_name": data[0], "last_name": data[1]}
 
 
 def main() -> QuerySet:
-    pass
+    genres_to_create = ["Western", "Action", "Dramma"]
+    for curr_item in genres_to_create:
+        Genre.objects.create(
+            name=curr_item
+        )
+
+    actors_to_create = [
+        get_firstname_lastname_from_full_name("George Klooney"),
```

Comment:
You could create list of tuples here
```suggestion
    actors_to_create = [
        ("George", "Klooney"),
```